CMake/resolve_dependency_modules/simdjson.cmake

```diff
@@ -33,4 +33,5 @@ if(${VELOX_SIMDJSON_SKIPUTF8VALIDATION})
   set(SIMDJSON_SKIPUTF8VALIDATION ON)
 endif()
 
+add_definitions(-DSIMDJSON_EXPERIMENTAL_ALLOW_INCOMPLETE_JSON)
```

Comment:
Please add this to the `simdjson::simdjson` target, no need to polute all compiler invocations with this definition.

Reply:
After makeavailalbe add `target_compile_definitions(simdjson::simdjson PUBLIC SIMDJSON_EXPERIMENTAL_ALLOW_INCOMPLETE_JSON)`
[Docs](https://cmake.org/cmake/help/latest/command/target_compile_definitions.html#command:target_compile_definitions) We need to add it as public because the option is used in the public header and not only at compile time of simdjson.